README.md

```diff
@@ -18,75 +17,95 @@ npm start
 
 Then open [`localhost:3001`](localhost:3001) in a browser.
 
+For more details see the source code of [examples](https://github.com/and-who/react-p5-wrapper/tree/master/example/src).
```

Comment:
Update to "of the examples" or "of our examples" instead of "of examples".

Reply:
Done!